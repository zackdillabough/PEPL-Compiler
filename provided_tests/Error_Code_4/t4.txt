POLY F1 = x + 1;
POLY F4 = x^2 + x + 4;
POLY F3(X, Y) = X - 1;
START
INPUT X;
F4(X);
F1(0);
F3(1, F1(0));
F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3(F4(F3
(F4(F3(F4(F3(F4(F3(X))))))))))))))))))));
363 2 2 1 0
