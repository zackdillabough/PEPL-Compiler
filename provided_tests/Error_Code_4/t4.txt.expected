Error Code 4: 9 9 9 9 9 9 9 10 10 10 